src/langcheck/augment/en/_justify_request.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import annotations
+
+from langcheck.metrics.eval_clients import (
+    EvalClient,
+)
+
+
+def justify_request(
```

Comment:
Maybe we could call this something like `rephrase_with_user_role_context`? And similarly, we could call the "roleplay" aug `rephrase_with_system_role_context`?

Docstrings for each could be
```
# For rephrase_with_user_role_context

    Rephrases each prompt in instances (usually a list of prompts) by adding the specified
    user role as context to each prompt. This adds context about the role of the user that is
    making the request.
    
    For example, if the prompt is "What is the capital of France?" and the role is
    "student", the augmented prompt might be "I'm a student working on my geography
    homework. What is the capital of France?"
```

```
# For rephrase_with_system_role_context

    Rephrases each prompt in instances (usually a list of prompts) by adding the specified
    system role as context to each prompt. This adds context about what role the AI should
    assume when responding.
    
    For example, if the prompt is "What is the capital of France?" and the role is
    "teacher", the augmented prompt might be "You are a teacher and you need to teach
    geography to your students. Now answer the query: What is the capital of France?"
```

Reply:
Makes sense, updated in [d5bfa2b](https://github.com/citadel-ai/langcheck/pull/170/commits/d5bfa2b28e94327db7213bafcb3d2e05a1c855b3)!!